v21.1/show-full-table-scans.md

```diff
@@ -0,0 +1,101 @@
+---
+title: SHOW FULL TABLE SCANS
+summary: The SHOW FULL TABLE SCANS statement lists recent queries that used a full table scan.
+toc: true
+---
+
+<span class="version-tag">New in v21.1:</span> The `SHOW FULL TABLE SCANS` [statement](sql-statements.html) lists recent queries for which CockroachDB performed a full table scan during query execution.
+
+Limiting the number of queries that require full table scans can help you optimize query execution performance. For more information on query performance optimization, see [Optimize Statement Performance](make-queries-fast.html) and [SQL Tuning with `EXPLAIN`](sql-tuning-with-explain.html).
+
+## Syntax
+
+<div>
+{% include {{ page.version.version }}/sql/generated/diagrams/show_full_scans.html %}
+</div>
+
+## Required privileges
+
+The [`admin` role](authorization.html#admin-role) is required to run `SHOW FULL TABLE SCANS`.
+
+## Examples
+
+To follow along, run [`cockroach demo`](cockroach-demo.html) to start a temporary, in-memory cluster with the sample [`movr` dataset](movr.html) preloaded:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach demo
+~~~
+
+Now, suppose that you want to query the `rides` table for all rides that cost above 90¤ (i.e., `WHERE revenue > 90`):
```

Comment:
@ericharmeling is `¤` a currency symbol?

Reply:
See my comment above

> ¤ is the universal currency sign. This is my attempt to be a little less US-centric with the docs (I think I've done this elsewhere as well).
>
>It could be more confusing than anything though - perhaps a docs style question, @mikeCRL ?

Reply:
I just removed the symbol since it's causing more confusion than anything.